packages/zosfiles/src/methods/download/Download.ts

```diff
@@ -298,7 +297,7 @@ export class Download {
 
             return {
                 success: true,
-                commandResponse: util.format(ZosFilesMessages.datasetDownloadedWithDestination.message, baseDir),
+                commandResponse: util.format(ZosFilesMessages.memberDownloadedWithDestination.message, baseDir),
```

Comment:
This change seems to be making a handful of system tests fail, in particular the zos-files package's `partitioned data set - all members` system tests. Please update the tests.

Reply:
That specific method is only downloading members not data sets. Originally, the message returned by it was "Data sets downloaded successfully", which I changed it to return "Member(s) downloaded successfully" since the method is downloading members.

Reply:
Thanks for the clarification. That does seem like the right message for the situation. The change still raises the question of whether it would be considered a breaking change. Perhaps the team should discuss in a parking lot if the message should be improved, or remain the same because it would be considered breaking change?